src/mlpack/core/data/data_options.hpp

```diff
@@ -320,6 +320,32 @@ class PlainDataOptions : public DataOptionsBase<PlainDataOptions>
 
 using DataOptions = PlainDataOptions;
 
+// Boolean options
+static const DataOptions Fatal   = DataOptions(true);
+static const DataOptions NoFatal = DataOptions(false);
+
+//! File options
+static const DataOptions CSV        = DataOptions(std::nullopt,
+    FileType::CSVASCII);
+static const DataOptions PGM_BIN    = DataOptions(std::nullopt,
```

Comment:
Do you think `PGM` would be a better name? Or `PGMBinary`?  The same for all the other formats. Maybe `CoordAscii` or `CoordASCII` instead of `COORD_ASCII`?  I guess all I mean is it makes sense to use caps for an acronym but no need for just words.